pkg/activator/throttler_test.go

```diff
@@ -128,6 +130,64 @@ func TestThrottler_UpdateCapacity(t *testing.T) {
 	}
 }
 
+func TestThrottler_ManyActivatorEndpoints(t *testing.T) {
+	const updatePollInterval = 10 * time.Millisecond
+	const updatePollTimeout = 3 * time.Second
+	activatorCount := 1
```

Comment:
This should test at least two activators, no?

Reply:
Nice catch! Updated this